src/devices/Camera/Camera/CameraInfo.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Camera;
+
+/// <summary>
+/// Information on the camera listed by the tools
+/// </summary>
+public record class CameraInfo(
```

Comment:
you will hate me but you're missing the intellisense comments for the parameters here.

Reply:
Sorry, I really missed this (and the other ones)!

Reply:
Don't be sorry :-) It happens and PR review are done to find those little things!